components/dashboard/src/teams/SSO.tsx

```diff
@@ -85,11 +90,10 @@ function OIDCClients(props: { organizationId: string }) {
             {modal?.mode === "edit" && <></>}
             {modal?.mode === "delete" && <></>}
 
+            <Heading2>OpenID Connect providers</Heading2>
```

Comment:
@selfcontained, here we've to be clear what's actually being set up, i.e. be clear about client vs. provider. Technically, we are not setting up a provider, but configuring a client for external providers. As there can be more than one client for a IdP (provider like `google`) we should avoid calling it provider here. 
Also, there are ideas floating to make Gitpod an IdP, even if that would be just proxying other IdPs, having the roles clarified prevents from confusion.

Let's go with "OpenID Connect clients" 🙏🏻